lib/parse-all.js

```diff
@@ -22,7 +23,10 @@ function parseAll(config) {
 					pattern: config.parsers[name].pattern
 				}))
 				.then(res => parsing[name].resolve(res))
-				.catch(err => parsing[name].reject(err));
+				.catch(err => {
+					parsing[name].reject(err);
+					return { name: name, error: err };
```

Comment:
We need to run all the parsers before we can generate error reporting. this way, we store which parser fails (`name`) and why (`error`)